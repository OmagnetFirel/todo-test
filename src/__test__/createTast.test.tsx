import React from 'react';
import renderer from 'react-test-renderer';
import CriarTarefa from '../components/CreateTask';

describe('CriarTarefa', () => {
    it('should render correctly', () => {
        console.log("teste", CriarTarefa)
        const component = renderer.create(<CriarTarefa />).toJSON();
        expect(component).toMatchSnapshot();
    });
})
