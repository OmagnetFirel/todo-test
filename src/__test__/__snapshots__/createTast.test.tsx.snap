// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CriarTarefa should render correctly 1`] = `
<div>
  <h1>
    Create Task
  </h1>
  <form>
    <input
      placeholder="Adicionar tarefa"
      type="text"
    />
    <button
      type="submit"
    >
      Adicionar
    </button>
  </form>
</div>
`;
